package.json

```diff
@@ -24,7 +24,8 @@
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.10.6",
     "morgan": "^1.10.0",
-    "multer": "^1.4.2"
+    "multer": "^1.4.2",
+    "react-validation": "^3.0.7"
```

Comment:
Dont change any files outside the frontend folder. Revert this and package.lock.json